recipe/run_test.py

```diff
@@ -1,17 +1,37 @@
 # Configure CuPy to use 1 GPU for testing
 import os
+import site
+import sys
 os.environ["CUPY_TEST_GPU_LIMIT"] = "1"
 
 # Check CUDA_PATH is set
 cuda_path = os.environ.get('CUDA_PATH')
 assert cuda_path is not None
 print("CUDA_PATH:", cuda_path)
 
-# Now that conda-forge docker images have libcuda.so, so "import cupy" would not fail.
-# However, tests would fail on the Azure CI since there is no GPU. See the discussion
-# in https://github.com/conda-forge/cupy-feedstock/pull/59#issuecomment-629584090
-import sys
-import cupy
+# check _wheel.json exists
+# note that we don't rely on cupy.__file__ because we cannot import CuPy on Windows
+pypaths = site.getsitepackages()
+for cp in pypaths:
+    config_json = os.path.join(cp, 'cupy/.data/_wheel.json')
+    if not os.path.isfile(config_json):
+        continue
+    else:
+        print("_wheel.json is found:", config_json)
+        break
+else:
+    raise RuntimeError('_wheel.json is not found in the package')
+
+# Now that conda-forge docker images have libcuda.so, so "import cupy" would not fail
+# on Linux. However, tests would fail on the Azure CI since there is no GPU. See the
+# discussion in https://github.com/conda-forge/cupy-feedstock/pull/59#issuecomment-629584090.
+# On Windows, this will fail because Windows has no driver stub.
+try:
+    import cupy
+except Exception as e:
+    if sys.platform.startswith('win32'):
+        print("No driver available. Exiting...")
```

Comment:
Should we mention Windows as well?

```suggestion
        print("No driver available on Windows. Exiting...")
```

Reply:
Done in 8e1564d.